app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,132 @@
+          </div>
+          <div class = "mb-2">
+            <%= label_tag :task_category, "Task Category", class: "form-label" %>
+            <%= select_tag :task_category, raw("<option>Inventory</option><option>Register</option>"), :class => "form-control" %>
+          </div>
+          <div class = "d-flex justify-content-center">
+            <%= label_tag nil, "Task Priority", class: "form-label" %>
+          </div>
+          <div class = "mb-3 d-flex justify-content-center">
+            <div class = "form-check form-check-inline">
+              <%= radio_button_tag :priority, 1, false, class: "form-check-input" %>
+              <%= label_tag :priority_1, 1, class: "form-check-label" %>
+            </div>
+            <div class = "form-check form-check-inline">
+              <%= radio_button_tag :priority, 2, false, class: "form-check-input" %>
+              <%= label_tag :priority_2, 2, class: "form-check-label" %>
+            </div>
+            <div class = "form-check form-check-inline">
+              <%= radio_button_tag :priority, 3, false, class: "form-check-input" %>
+              <%= label_tag :priority_3, 3, class: "form-check-label" %>
+            </div>
+            <div class = "form-check form-check-inline">
+              <%= radio_button_tag :priority, 4, false, class: "form-check-input" %>
+              <%= label_tag :priority_4, 4, class: "form-check-label" %>
+            </div>
+            <div class = "form-check form-check-inline">
+              <%= radio_button_tag :priority, 5, false, class: "form-check-input" %>
+              <%= label_tag :priority_5, 5, class: "form-check-label" %>
+            </div>
+          </div>
+          <div class = "mb-2 d-flex justify-content-center">
+            <%= submit_tag "Add Task +", :class => "btn btn-danger" %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Tasks div -->
+<div id = "tasks-div" class = "container">
+  <!-- Task panel -->
+  <div class = "task-panel row px-1 py-1 border-bottom border-top border-dark">
+    <!-- Checkbox -->
+    <div class = "col-1 d-flex flex-column justify-content-center align-items-center">
+      <%= check_box_tag :checkoff_task %>
+    </div>
+    
+    <!-- Right side of panel -->
+    <div class = "col-11 ps-4">
+      <div class = "d-flex justify-content-between align-items-center">
+        <h5 class = "fw-bold m-0">This is a task!</h5>
+        <span class="badge bg-success">Inventory</span>
+      </div>
+      <p class = "fw-lighter small m-0"><small>Completed by Roberto at 10:10am 12/12/12</small></p>
+      <p class = "fw-lighter small m-0"><small>Posted by Melody 9:00am 12/12/12</small></p>
+      <p class = "m-0">
+        <small>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
um, you should probably remove the "greeking" from customer-visible views at this point